.github/workflows/golangci-lint.yml

```
@@ -27,7 +27,7 @@ jobs:
         uses: golangci/golangci-lint-action@2e788936b09dd82dc280e845628a40d2ba6b204c # v6.3.1
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.57.1
+          version: v1.63.4
```

Should we set this to the latest version v1.64.5?

---

I just confirmed this. Please refer to https://github.com/etcd-io/raft/pull/268#issuecomment-2670649123

```suggestion
          version: v1.64.5
```